### ISSUE TITLE
Paginate output  from the help command

### ISSUE THREAD

**Issue by CharliePoole** [nunit/nunit-console#1480]
The help screen is getting very long. While it's possible to send the output to `more`, that loses the color display because of how our implementation of  `ColorConsole` works. So I'd like to create an internal implementation that looks like more, but without a an of the options it supports.

1. Initial output will fill the screen with `-- More  --` on the last line.
2. Pressing enter will advance one line
3. Pressing space will advance one page
4. Pressing 'q' or 'Q' will exit.

**Comment by veleek** [comment]
IMO I don't think this is a problem that we should be solving with the runner.  The color in the output only provides a marginal benefit (we're highlighting the headers in one color and all of the body text is in green, there's no syntax highlighting or anything like that to make things clearer) and I think it's better to just support whatever default pagination method the platform provides.

I would support either:
1. Add a "simple" help page that outputs by default and provide additional arguments for more detailed output.
2. Determine if there's a way to support colored console output that doesn't break when piping to a pager.

**Comment by CharliePoole** [comment]
Yeah. I think you're right about the pagination. Maybe a short page displayed for simple --help or --help:usage and a longer one for --help:options, etc.
